modules/acquia_cms_headless/acquia_cms_headless.install

```diff
@@ -33,3 +39,49 @@ function acquia_cms_headless_requirements($phase) {
 
   return $requirements;
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * Update all consumers and assign content model roles and add
+ * provide `issue subrequests` permission to authenticated & anonymous role.
+ */
+function acquia_cms_headless_update_8001() {
+  $rolesStorage = \Drupal::entityTypeManager()->getStorage('user_role');
+  $query = $rolesStorage->getQuery();
+  $roleIds = $query
+    ->condition('id', ["authenticated", "anonymous", "headless"], 'NOT IN')
+    ->execute();
+
+  $consumerStorage = \Drupal::entityTypeManager()->getStorage('consumer');
+  $query = $consumerStorage->getQuery();
+  $consumerIds = $query
+    ->exists('user_id')
+    ->execute();
+  $consumers = $consumerStorage->loadMultiple($consumerIds);
+  foreach ($consumers as $consumer) {
+    $user = $consumer->user_id->entity;
+    foreach ($roleIds as $roleId) {
+      $user->addRole($roleId);
+    }
+    $user->save();
```

Comment:
will this also impact more consumers/users than we expecting? What if a customer site has a consumer for specific purpose and this change will add admin role/scope to that